packages/cli/src/logging/logger.service.ts

```diff
@@ -81,11 +104,25 @@ export class Logger {
 		this.internalLogger.add(new winston.transports.Console({ format }));
 	}
 
+	private scopeFilter() {
+		return winston.format((info: TransformableInfo & { metadata: LogMetadata }) => {
+			if (
+				this.isScopingEnabled &&
+				(!info.metadata.scope || !this.scopes.has(info.metadata.scope))
```

Comment:
A bit convoluted. Could be made more readable by extracting the scope part:

```suggestion
			const shouldIncludeScope = info.metadata.scope && this.scopes.has(info.metadata.scope)
			if (this.isScopingEnabled && !shouldIncludeScope)
```